tmt/schemas/provision/hardware.yaml

```
@@ -145,6 +145,32 @@ definitions:
     items:
       "$ref": "#/definitions/network"
 
+  # HW requirements: single `system` item
+  system:
+    type: object
+
+    properties:
+      vendor:
+        type: string
+
+      model:
+        type: string
+
+      numa-nodes:
+        type: string
```

```suggestion
        type: integer
```

This should always be a number, right?

---

It should be both, an integer (`numa-nodes: 4`) and a string (`numa-nodes: ">= 4"`).